pages/data-design/defining-descriptive-events-and-properties.mdx

```diff
@@ -23,8 +23,11 @@ selecting or changing event and property names. Note that some analytics tools
 require specific casing for events or properties.
 
 <center>
-  <img
-    src={require('../../images/best-practices/descriptive-events-and-properties/naming-assistant.gif')}
+  <video
+    loop
+    muted
+    autoPlay
```

Comment:
I would suggest adding `controls` here as many browsers block autoplaying and in this case the user has to right-click the video (if they realize it's a video at all) and choose Play.

Reply:
Oops, thought I had put it on all of them but apparently missed this one, fixed in 62eae3d